chainer/functions/loss/ctc.py

```diff
@@ -293,23 +293,32 @@ def connectionist_temporal_classification(
 
     Args:
         x (list or tuple of :class:`~chainer.Variable`):
-            RNN output at each time. Each element of ``x``, ``x[i]``
-            is a :class:`~chainer.Variable` representing output of RNN at time
-            ``i``.
+            A list of unnormalized probabilities for labels.
+            Each element of ``x``, ``x[i]`` is a :class:`~chainer.Variable`
+            object, which has shape ``(B, V)``, where ``B``
+            is the batch size and ``V`` is the number of labels.
+            The softmax of ``x[i]`` represents the probabilities of the labels
+            at time ``i``.
         t (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
         :class:`cupy.ndarray`):
             Expected label sequence.
```

Comment:
label sequences? (should it be plural?)

Reply:
I think there's only one series of labels, so one sequence should be enough to label them, so stay with `label sequence`.

Reply:
Ah, it is a matrix that includes various label sequences. How about "a matrix including expected label sequences"?

Reply:
That's better. Could you update with the sentence?